baselines/doc/source/index.rst

```diff
@@ -25,6 +25,8 @@ Flower Baselines are a collection of organised scripts used to reproduce results
    :maxdepth: 1
    :caption: Flower Baselines
    
-   using-baselines
-   contributing-baselines
+   tutorials
+   guides
+   reference
+   explanations
```

Comment:
@charlesbvll is it possible to start in expanded mode?

Reply:
@tanertopal No, unfortunately, this is something that Daniel asked me as well but it is not possible (except if we fork the Sphinx project or the Furo theme)

Reply:
@charlesbvll do you know what would need to be changed in Furo? Afaik it's easy to override template files.

Reply:
@tanertopal This is the approach I attempted but when modifying the template file I could expand it but then the collapsing functionality didn't work anymore